### DISCUSSION TITLE
ExoPlayer2 with Expo for low latency live streaming.

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#5689] [General]
Hi folks, With the Expo app how should I proceed to further leverage low-latency streaming? I am willing to use Ant Media for streaming, as they provide SDK for React Native, not sure if any one of you guys worked with ExoPlayer + ExoPlayer + Ant Media React Native SDK.

Link for SDK: https://github.com/ant-media/WebRTC-React-Native-SDK

Please help me with the flow or architecture I should focus on.

> Source: [Reddit](https://www.reddit.com/r/expo/comments/147hf8h/exoplayer2_with_expo_for_low_latency_live/)

**Comment by mekya** [comment]
Hi @Mohit-3196,

There is an issue created here for this as well: https://github.com/ant-media/Ant-Media-Server/issues/5746

I think we can schedule this implementation according to priority.

Priority can be assigned according to the following items:
- If it's coming from an Enterprise user and/or how many users are asking this feature
  - The amount of usage of the Enterprise user
- If there is any new opportunity
- If it's really important to develop this feature

**Reply by Mohit-3196** [reply]
Thank you for your inputs abi.
